include/fxcg/display.h

```diff
@@ -87,6 +87,16 @@ void Bdisp_ShapeBase3XVRAM( void*shape );
 void Bdisp_ShapeBase( unsigned char*work, struct display_shape *shape, int color, int line_width, int zero1, int zero2 );
 void Bdisp_ShapeToVRAM16C( void*, int color );
 void Bdisp_ShapeToDD( void*shape, int color );
+
+// Draws rectangle to VRAM, x between 0 and 383 (inclusive), y between 0 and 191 (inclusive).
+// These avoid the status bar:
```

Comment:
Does this apply to all three functions? Please clarify.

Reply:
And for that matter, what does it mean to "avoid the status bar"?

Reply:
There is more information on [WikiPrizm](https://prizm.cemetech.net/index.php?title=Bdisp_Rectangle), but I have clarified the statements in the code.